calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -67,253 +67,220 @@ Enabling WAF for certain system services may result in an undesired cluster stat
 
-3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
+```bash
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/release/v0.7.0/release/kubernetes-manifests.yaml
+```
 
-4. To make further changes to a service, click **Actions**, and then **Enable** or **Disable**.
+#### Enable WAF using kubectl CLIs
 
+In the ApplicationLayer custom resource, named `tigera-secure`, set the `webApplicationFirewall` field to `Enabled`.
 
-### View WAF events
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: ApplicationLayer
+metadata:
+  name: tigera-secure
+spec:
+  webApplicationFirewall: Enabled
+EOF
+```
 
-#### Service Graph
+### Enable WAF using Manager UI
 
-To view WAF events in Service Graph,
+On the Manager UI, click **Threat Defense**, **Web Application Firewall**, **Configure Web Application Firewall**.
 
-1. Select a service.
-2. Click the HTTP tab.
+ <img
+    src='/img/calico-enterprise/waf-config-first-step.png'
+    alt='WAF services'
+    width='600'
+  />
 
-To simulate an attack to view the default warning in Service Graph, follow these steps:
+:::note
 
-1. Get the cluster/service URL.
-   - The external address of your cluster/service
-   - The cluster IP of your application's service (if testing within the cluster)
-2. Enter the following command to trigger a SQL injection attack.
+If WAF is already enabled in your cluster, you do not see this page.
+:::
 
-##### Example: SQL Injection attack
+### Apply WAF to services
 
-```
-  curl http://<host>//test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user
-```
+Now that your cluster is enabled for WAF, you can apply WAF to your services and make sure that your
+Web apps are protected from application layer attacks. 
 
-In Service Graph, go to the service and click the HTTP tab. You will see the following warning:
+If you have deployed the sample application, you can apply WAF on a service associated with your app, as follows:
 
+```bash
+kubectl annotate svc frontend -n default --overwrite projectcalico.org/l7-logging=true
 ```
-  level=warning msg="WAF Process Http Request [2f435aca-4a3e-4f96-a7b2-a60c9745bd7b] URL '/test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user' OWASP Warning'[2] Host:'10.224.0.38' File:'/etc/modsecurity-ruleset/REQUEST-942-APPLICATION-ATTACK-SQLI.conf' Line:'45' ID:'942100' Data:'' Severity:'0' Version:'OWASP_CRS/3.3.2' Message:'Warning. detected SQLi using libinjection.''"
-```
+Alternatively, you can use the Manager UI to apply WAF to the `frontend` service.
 
-#### Kibana
+In this example, we applied WAF to the `frontend` service. This means that every traffic that goes through the `frontend` service is inspected.
```

Comment:
"This means that every traffic that goes through" - I think this would read better as "This means that every request that goes through"

Reply:
Fixed.